### DISCUSSION TITLE
Embedded script not detected

### DISCUSSION THREAD

**Discussion by seqizz** [numToStr/Comment.nvim#323] [Q&A] [answered]
Hi :wave: 

First, thanks for the plugin. Works a lot better than alternatives.

I'd like to ask for a very minor gimmick: I have a mixed vim config file, which I embed lua like:

```
lua << EOF
```

And plugin (probably rightfully) ignores this and puts comments in this section also like parent file type (`"`). Any way to workaround these stuff?

**Comment by numToStr** [comment, marked as answer]
You might need to install vimscrip and lua treesitter parser. After that, commenting should work properly inside heredoc.

```
:TSInstall vim lua
```
